### PR TITLE
docs: add logical mask example

### PR DESCRIPTION
**Pull request by JoepVanlier**
**Why this PR?**
This has come up in support twice now. Might as well add a sentence or two to show people how to combine different criteria when boolean array indexing the data. Also noticed some deprecated import location in `scipy` that I fixed up.

Docs build here: https://lumicks-pylake.readthedocs.io/en/add_logic_example/tutorial/fdfitting.html#adding-data-to-the-fit

### CODE REVIEW

**Review by aafkevandenberg — APPROVED** [review]
Nice, very useful addition! I think just giving the succinct notation when compairing 3 conditions would be sufficient. That form is also easier to extent to even more conditions.

File: docs/tutorial/fdfitting.rst [hunk 1]
```diff
@@ -174,6 +174,14 @@ when loading the data from `DNA/Lc` to `DNA/Lc_RecA`::
...
     fit.add_data("RecA", force2, distance2, params={"DNA/Lc": "DNA/Lc_RecA"})
 
+Multiple criteria can be combined into a single mask by using `numpy`'s logical operators.
+For example, if we want to select all forces up to `30` pN, and only include distances between `1.5` and `4.0` micron, we could create the mask by invoking::
+
+    example_mask = np.logical_and(np.logical_and(fd1.f.data <= 30, fd1.d.data > 1.5), fd1.d.data < 4.0)
+
+Or more succinctly (note the double brackets)::
+
+    example_mask = np.logical_and.reduce((fd1.f.data <= 30, fd1.d.data > 1.5, fd1.d.data < 4.0))
```
Comment (aafkevandenberg) [inline docs/tutorial/fdfitting.rst:184]: Multiple criteria can be combined into a single mask by using `numpy`'s logical operators.
For example, if we want to select all forces up to `30` pN and distance data larger than '1.5' micron: 

    example_mask = np.logical_and(d1.f.data <= 30, fd1.d.data > 1.5)

Adding a third condition that the distance has to be < 4 micron (note the double brackets)::

    example_mask = np.logical_and.reduce((fd1.f.data <= 30, fd1.d.data > 1.5, fd1.d.data < 4.0))

Comment (JoepVanlier) [reply docs/tutorial/fdfitting.rst:184]: Good point! Amending!

File: lumicks/pylake/force_calibration/detail/driving_input.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import numpy as np
-import scipy.signal
+import scipy.signal.windows
```
Comment (tobiasjj) [inline lumicks/pylake/force_calibration/detail/driving_input.py:2]: 👍

**Review by tobiasjj — APPROVED** [review]
LGTM! 🚴

**Pull request merged**